locale/en/labels.cfg

```diff
@@ -127,13 +127,13 @@ of 20 municipalities.
 109 inhabitants per km².
 '''
     [[reg_mun_gen_cap_re]]
-        title = 'Installed Capacity Renewables'
+        title = 'Installed renewable Capacity'
```

Comment:
Sound better.
But the capitalization is fine to my mind since the layer names are titles and capitals support readability, cf. [1](https://www.dailywritingtips.com/rules-for-capitalization-in-titles/), [2](https://grammar.yourdictionary.com/capitalization/rules-for-capitalization-in-titles.html)

(applied to the other changed lines as well)

Reply:
U wanna change it? I can do it too..

Reply:
ok - so should I do it? Or is it already done?

Reply:
Go ahead ;)

Reply:
I'll do it now